src/index.js

```diff
@@ -36,6 +53,12 @@ class CreateDynamoDBGlobalTables {
   }
 
   async createGlobalTable(tableName, region) {
+    const tableHasBeenUpgraded = await this.tableIsAtVersionNumber(tableName);
+    if (tableHasBeenUpgraded) {
+      this.log(`Table ${tableName} has been upgraded to verision ${GLOBAL_TABLE_VERSION}`);
+
+      return false;
+    }
     const dynamo = new AWS.DynamoDB({ region });
     const params = {
```

Comment:
in the docs https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_CreateGlobalTable.html it says that the `createGlobalTable` operation below is only applicable for `[Version 2017.11.29]`